src/main/java/com/microsoft/graph/requests/extensions/AppRoleAssignmentCollectionReferenceRequest.java

```diff
@@ -38,15 +38,15 @@ public AppRoleAssignmentCollectionReferenceRequest(final String requestUrl, IBas
 
     public void post(final AppRoleAssignment newAppRoleAssignment, final ICallback<? super AppRoleAssignment> callback) {
         final String requestUrl = getBaseRequest().getRequestUrl().toString();
-        final ReferenceRequestBody body = new ReferenceRequestBody(getBaseRequest().getClient().getServiceRoot() + "/me/ownersAsAppRoleAssignment/" + newAppRoleAssignment.id);
+        final ReferenceRequestBody body = new ReferenceRequestBody(getBaseRequest().getClient().getServiceRoot() + "/me/registeredUsersAsAppRoleAssignment/" + newAppRoleAssignment.id);
```

Comment:
@zengin @MIchaelMainer I'm not sure where this is coming from, none of those endpoints seem to exist, unless some private preview kind of leaked in the public metadata?

Reply:
This looks like influence from the cast/filter functionality. Neither of these URLs are correct. This requires an investigation.

Reply:
created #562